src/main/java/picard/illumina/parser/IlluminaFileUtil.java

```diff
@@ -201,7 +201,14 @@ public ParameterizedFileUtil getUtil(final SupportedIlluminaFormat format) {
 
         final List<Integer> tiles = getUtil(formats.get(0)).getTiles();
         for (int i = 1; i < formats.size(); i++) {
-            final List<Integer> fmTiles = getUtil(formats.get(i)).getTiles();
+            ParameterizedFileUtil fileUtil = getUtil(formats.get(i));
+
+            //for run based files we don't care about tile matching.
+            if(fileUtil instanceof PerTileOrPerRunFileUtil && ((PerTileOrPerRunFileUtil)fileUtil).getRunFile() != null)
```

Comment:
Need space after `if` also `{` should be at end of line. Also `//` comments should have a space before the first character, and sentences should be capitalized.